src/Server/Repositories/ClaraJobRepository.cs

```diff
@@ -0,0 +1,236 @@
+﻿/*
+ * Apache License, Version 2.0
+ * Copyright 2019-2021 NVIDIA Corporation
```

Comment:
is this all new code? then the copyright should be 2021 and not 2019-2021

Reply:
Nope, renamed.